### PR TITLE
Base: Added manpages for which(1) and stat(1)

### PR DESCRIPTION
**Pull request by jahway603**
Added manpages for which(1) and stat(1)

### CODE REVIEW

File: Base/usr/share/man/man1/stat.md
Comment (networkException) [inline Base/usr/share/man/man1/stat.md:35]: I'd say lets go with what serenity actually outputs here

```suggestion
$ stat -L /bin/ls
    File: /bin/ls
   Inode: 8288
    Size: 184896
   Links: 1
  Blocks: 376
     UID: 0 (root)
     GID: 0 (root)
    Mode: (100755/-rwxr-xr-x)
Accessed: 2022-10-20 03:56:56
Modified: 2022-10-20 03:55:34
 Changed: 2022-10-20 03:56:57
```

Comment (jahway603) [reply Base/usr/share/man/man1/stat.md:35]: Alright, I manually typed in what I saw in Serenity, but will keep aware of the spacing of the output.

Comment (jahway603) [reply Base/usr/share/man/man1/stat.md:35]: Alright, I manually typed in what I saw in Serenity, but will keep aware of the spacing of the output. What do I need to do to fix this so that this PR can be merged?

Comment (networkException) [reply Base/usr/share/man/man1/stat.md:35]: If you edit the codeblock to be the exact output of the command (including leading spaces) I will approve the pr and link it on discord so a maintainer can take a look. Although you are encouraged to join the discord and advertise the pr yourself

**Comment by jahway603** [comment]
@networkException is the "pr-waiting-for-author" tag an indication for me to do something else on this PR?

**Comment by networkException** [comment]
Yes I left another review requesting a chnage

**Review by networkException — CHANGES_REQUESTED** [review]

**Comment by networkException** [comment]
Oh my bad, GitHub mobile didn't show this as pending

**Comment by jahway603** [comment]
@networkException fixed that and squashed the commits. Anything I need to do to get this PR merged?

**Review by networkException — APPROVED** [review]

File: Base/usr/share/man/man1/stat.md
Comment (linusg) [inline Base/usr/share/man/man1/stat.md:15]: I realize that the colon is present in a couple of man pages already, but headings should never have one (which the majority of man pages agrees with). Please remove it.

File: Base/usr/share/man/man1/stat.md
Comment (linusg) [inline Base/usr/share/man/man1/stat.md:3]: This is missing a summary

File: Base/usr/share/man/man1/stat.md
Comment (linusg) [inline Base/usr/share/man/man1/stat.md:13]: This could be used as the summary and have a little bit more information here instead.

File: Base/usr/share/man/man1/which.md
Comment (linusg) [inline Base/usr/share/man/man1/which.md:3]: This should be in imperative mood, `show ...`

File: Base/usr/share/man/man1/which.md
Comment (linusg) [inline Base/usr/share/man/man1/which.md:13]: This is clearly copied from the Linux man page (https://linux.die.net/man/1/which), including the awkward wording of "prints to stdout the full path of ..." (instead of the correct "prints the full path of ... to stdout"). Please don't do that.

**Review by linusg — CHANGES_REQUESTED** [review]

**Comment by linusg** [comment]
https://github.com/SerenityOS/serenity/blob/master/CONTRIBUTING.md#code-submission-policy

> Write the commit message subject line in the imperative mood ("Foo: Change the way dates work", not "Foo: Changed the way dates work").

**Comment by jahway603** [comment]
> https://github.com/SerenityOS/serenity/blob/master/CONTRIBUTING.md#code-submission-policy
> 
> > Write the commit message subject line in the imperative mood ("Foo: Change the way dates work", not "Foo: Changed the way dates work").

Any other changes that are needed before this PR is merged?

**Comment by jahway603** [comment]
@networkException does this PR need to be "re-advertised" or what's the nest step here?

File: Base/usr/share/man/man1/which.md
Comment (linusg) [inline Base/usr/share/man/man1/which.md:13]: Still copied from the Linux man page, except you cut some words off so it no longer even makes sense:

> full path of the executables that would have been executed when this argument **_had been entered_ at the shell prompt.**

Delete this and write it in your own words.

**Review by linusg — CHANGES_REQUESTED** [review]

**Comment by jahway603** [comment]
@linusg what else needs to be resolved for this PR to be merged?

File: Base/usr/share/man/man1/which.md
Comment (jahway603) [reply Base/usr/share/man/man1/which.md]: done

**Pull request merged**
